include/experimental/__p0009_bits/layout_left.hpp

```diff
@@ -110,9 +110,12 @@ struct layout_left {
     MDSPAN_TEMPLATE_REQUIRES(
       class OtherExtents,
       /* requires */ (
-        _MDSPAN_TRAIT(is_convertible, OtherExtents, Extents)
+        _MDSPAN_TRAIT(is_constructible, OtherExtents, Extents)
       )
     )
+#ifdef _MDSPAN_USE_CONDITIONAL_EXPLICIT
+    explicit (!is_convertible<OtherExtents, Extents>::value)
+#endif
```

Comment:
```suggestion
    _MDSPAN_CONDITIONAL_EXPLICIT (!is_convertible<OtherExtents, Extents>::value)
```

Reply:
Would the comma inside a macro invocation need protection (by wrapping the whole expression in an extra pair of parens)?

Reply:
Probably so